src/templates/landing_a.js

```diff
@@ -43,7 +55,7 @@ const Landing = (props) => {
 
   // data sent to the form already prefilled
   const preData = {
-    course: {type: "hidden", value: yml.meta_info.utm_course, valid: true},
+    course: {type: "hidden", value: `${programs <=1 ? (programs[0].value) : (yml.meta_info.utm_course)}`, valid: true},
```

Comment:
when the length of the array of utm_course is 1 it returns the corresponding value, otherwise it returns the default utm_course array